src/components/output/custom-els/MultiPanel/index.ts

```diff
@@ -0,0 +1,220 @@
+import './styles.css';
+
+/**
+ * A multi-panel view that the user can add any number of 'panels'.
+ * 'a panel' consists of two elements. Even index element becomes heading,
+ * and odd index element becomes the expandable content.
+ */
```

Comment:
Move this comment above the class - that means things like VSCode will show it along with the type hinting.